ReactWindows/ReactNative.Net46/UIManager/UIManagerModule.cs

```diff
@@ -77,7 +84,7 @@ public override string Name
         /// <summary>
         /// The event dispatcher for the module.
         /// </summary>
-        public EventDispatcher EventDispatcher
+        public IEventDispatcher EventDispatcher
```

Comment:
Added an interface to EventDispatcher to allow limited access to certain methods needed by components that need to listen to the EventDispatcher...